winml/lib/Api/LearningModelSession.h

```diff
@@ -87,7 +87,13 @@ struct LearningModelSession : LearningModelSessionT<LearningModelSession, ILearn
     return &dml_ep_lock_;
   }
 
-  static winml::LearningModelSession CreateInertSession(_winml::IEngine* engine);
+  static
+  winml::LearningModelSession
+  CreateInertSession(
+    _winml::IEngine* engine);
```

Comment:
Why was this formatting changed? Is it automatically formatted from a cmake format file? It looks kind of odd and overly verbose.

Reply:
used in other places for consistency